wpengine/atlas-content-modeler
# BH-1143: Fix negative numbers min vs max

## Description

Update number type to not allow entering numbers less than min or more than max including handling negative numbers.

https://wpengine.atlassian.net/browse/BH-1143

## Testing

1. Check for number required
2. Check for number not required
3. Check for input less than min
4. Check for input more than max
